src/Core/Domain/Customer/Group/Command/AddCustomerGroupCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Domain\Customer\Group\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\ValueObject\GroupName;
+
+class AddCustomerGroupCommand
+{
+    /** @var string[] */
+    private $localizedNames;
```

Comment:
there you should probably use your VO GroupName (if you dcide to implement related assertions and still have the GroupName class)
```suggestion
    /** @var GroupName[] */
    private $localizedNames;
```

Reply:
I don't think so, this VO should be removed (as explained in a later comment)

Reply:
I deleted the GroupName VO